releases/release-1.12/release-1.12.md

```diff
@@ -0,0 +1,136 @@
+| Release retrospective | Community | | | | 4 | | | |
+
+## Details
+
+### Conditionally Shortened Code Freeze
+
+In an effort to make the Code Freeze period shorter, and allow more time for feature development, the 1.11 Release Team tested out a procedure for postponing it based on release stability and "clean signal".
+
+The goal is that all tests in [master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking) and [master-upgrade](https://k8s-testgrid.appspot.com/sig-release-master-upgrade) are passing and have been passing for a few days.
+
+Based on the 1.11 outcome, the 1.12 Release Team will make **TBD** changes to the schedule.
+
+### Feature Freeze
+
+All features going into the release must have an associated issue in the features repo by ***TBD July ??th***. That issue must be in the 1.12 milestone.  SIG "themes" should also be in the release notes draft at this time to prepare for blog posts and release marketing.  Any work the SIG wants publicized needs to be called out to the Features Lead so the Release Team communications lead can work with SIG-PM and the CNCF.
+
+### Code Slush
+
+Starting on ***TBD July ??nd***, only PRs labeled by their owner SIGs with [`status/approved-for-milestone`](/ephemera/issues.md) will be allowed to merge into the master branch. All others will be deferred until the end of Code Freeze, when master opens back up for the next release cycle. If necessary, the release team can add the `status/approved-for-milestone` label in cases where the SIG approvers do not have permissions to do so.
+
+At this time, any issues/PRs with incomplete labels, or which are "stale" will be removed from the release.  The release team will work with SIGs to make sure that labels are complete prior to removal.
+
+Code Slush begins prior to Code Freeze to help reduce noise from miscellaneous changes that aren't related to issues that SIGs have approved for the milestone. Feature work is still allowed at this point, but it must follow the process to get approved for the milestone. SIGs are the gatekeepers of this label, not the release team.
+
+#### Exceptions
+
+Starting at Code Slush, the release team will solicit and rule on [exception requests](https://github.com/kubernetes/features/blob/master/EXCEPTIONS.md) for feature and test work that is unlikely to be done by Code Freeze. As with the `status/approved-for-milestone` label, the exception approval is the responsibility of the SIG or SIGs labeled in the pull request. The release team may intervene or deny the request only if it poses a risk to release quality, or could negatively impact the overall timeline. Changes introduced at this point should be well-tested, well-understood, limited in architectural scope, and low risk.  All of those factors should be considered in the approval process.
+
+### Code Freeze
+
+All features going into the release must be code-complete (including tests) and have docs PRs open by ***TBD August ??th***.
```

Comment:
can we take this out of parentheses and bold it? plz consider markdown below

All features going into the release must be code-complete, **including tests**, and have docs PRs open by